lib/screens/login/view/init_screen.dart

```diff
@@ -49,14 +52,18 @@ class InitScreenState extends State<InitScreen> with TickerProviderStateMixin {
   }
 
   Future<Widget> loadInitialScreen() async {
-    ApiDatabase database = Locator.instance<ApiDatabase>();
-    WalletStorage storage = await database.loadWalletsDatabase();
+    try {
```

Comment:
Why are we using a `try catch` statement here to render the first time usage actions if an error is thrown? Is it necessary for the log file? If not, could you add this to a different commit?